Zend/tests/fibers/backtrace-deep-nesting.phpt

```diff
@@ -0,0 +1,51 @@
+--TEST--
+Backtrace in deeply nested function call
+--FILE--
+<?php
+
+function suspend_fiber(int $level): void {
+    if ($level >= 10) {
+        $value = \Fiber::suspend($level);
+        failing_function($value);
+    }
+
+    suspend_fiber($level + 1);
+}
+
+function failing_function(string $value): never {
+    throw_exception();
+}
+
+function throw_exception(): never {
+    throw new Exception;
+}
+
+$fiber = new Fiber(function (): void {
+    suspend_fiber(0);
+});
+
+$fiber->start();
+
+$fiber->resume('test');
+
+?>
+--EXPECTF--
+Fatal error: Uncaught Exception in %sbacktrace-deep-nesting.php:%d
+Stack trace:
+#0 %sbacktrace-deep-nesting.php(%d): throw_exception()
+#1 %sbacktrace-deep-nesting.php(%d): failing_function('test')
+#2 [suspended] %sbacktrace-deep-nesting.php(%d): suspend_fiber(10)
```

Comment:
When is `[suspended]` helpful, couldn't it be just analysed from the backtrace when needed?

Reply:
`[suspended]` is helpful for knowing what portion of the backtrace was executed after resuming at a glance. It's a suggestion, not strictly necessary.

The trace that resumed the fiber is included below the line marked with `{fiber}`.

Reply:
> The trace that resumed the fiber is included below the line marked with {fiber}.

then `[suspended]` is unnecessary/redundant information

Reply:
`[suspended]` is not redundant, it tags calls that were on the fiber stack before the last time the fiber was suspended. In this trace, `failing_function` and `throw_exception` were called after resuming, so they are not tagged with `[suspended]`.

`[was suspended]` or `[resumed]` may be better, I'm not exactly sure what makes the most sense without being too verbose while still providing clarity.

Reply:
Decided to drop this for now. I wasn't sold on the tag myself, I was worried it would be confusing and I think this conversation shows those concerns were justified. I'd rather focus on improving the backtraces with the start/resume/throw trace first.